proto/kvrpcpb.proto

```diff
@@ -766,6 +766,13 @@ enum IsolationLevel {
     RC = 1; // RC = read committed
 }
 
+//Operation allowed level during each TiKV disk full threshold.
```

Comment:
Generally there should be a space after `//`.

Reply:
done!